siliconcompiler/core.py

```diff
@@ -850,15 +850,14 @@ def help(self, *keypath):
         return fullstr
 
     ###########################################################################
-    def valid(self, *keypath, valid_keypaths=None, default_valid=False):
+    def valid(self, *keypath, default_valid=False):
         """
         Checks validity of a keypath.
 
         Checks the validity of a parameter keypath and returns True if the
         keypath is valid and False if invalid.
 
         Args:
-            keypath(list str): Variable length schema key list.
```

Comment:
Did you mean to remove this? (looks like the actual arg you deleted was just never documented...)